docs/sources/panels-visualizations/visualizations/pie-chart/index.md

```diff
@@ -81,15 +81,6 @@ The following example shows a pie chart with **Name** and **Percent** labels dis
 
 {{< docs/shared lookup="visualizations/legend-mode.md" source="grafana" version="<GRAFANA VERSION>" >}}
 
```

Comment:
Very reasonable assumption to make, but in this case, the text is more accurate than the shared content.
```suggestion
### Legend values

Select values to display in the legend. You can select more than one.

- **Percent:** The percentage of the whole.
- **Value:** The raw numerical value.

For more information about the legend, refer to [Configure a legend]{{< relref "../../configure-legend" >}}).

```